Comparison Engine/Comparison Engine/Base Classes/BarManager.cs

```diff
@@ -16,26 +17,26 @@ private BarManager()
         }
 
         //A list of bars
-        public List<Bar> barList = new List<Bar>();
+        public Dictionary<int, Bar> barDictionary  { get; set; }
```

Comment:
Does getting rid of new List<Bar>() not cause a null exception when executing the populateLists in Form1.cs, since you're writing to those variables, both barDictionary and drinkDictionary without allocating them?